pkg/controller/dataimportcron-controller.go

```diff
@@ -178,7 +178,7 @@ func (r *DataImportCronReconciler) initCron(ctx context.Context, dataImportCron
 		}
 		return nil
 	}
-	if !isURLSource(dataImportCron) {
+	if !isURLSource(dataImportCron) && !isHTTPSource(dataImportCron) {
```

Comment:
Names are not clear as they smell the same - Maybe isRegistryURLSource?

Reply:
> Names are not clear as they smell the same - Maybe isRegistryURLSource?

Sure.